webgui/src/flipmouse/js/tabslot.js

```diff
@@ -35,13 +83,95 @@ tabSlot.initSlots = function () {
     });
 };
 
+tabSlot.initProcesses = function () {
+	if(inElectron === false) {
+		var resultList = ["not available / nicht verfügbar"];
+		//add the process list to the GUI
+		L('.process-select').forEach(function (elem) {
+			elem.innerHTML = L.createSelectItems(resultList);
+		});
+		return;
+	}
+	
+	storage.setDataPath(require('os').homedir());
+	console.log("Looking for config json here: " + storage.getDataPath());
+	
+	//get an active process if one is set for this slot
+	storage.get('FLipMouseGUIConfiguration',function(error, processActivationList) {
+		//if we got a list back (JSON file exists & contains data)
```

Comment:
maybe also move reading of config to another file, maybe `electronUtil.js` that contains both methods to retrieve current running processes and methods to retrieve/save config.

Reply:
Is now in elecUtil.js (remember: 8.3 filenames :-))